src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -748,6 +761,82 @@ private JsValue IncrementCounter(JsValue self, JsValue[] args)
                 return JsBoolean.True;
             }
 
+            private JsValue InvokeTimeSeriesFunction(JsValue self, JsValue[] args)
+            {
+                // todo aviv : write meaningful error messages
+
+                AssertValidDatabaseContext();
+                if (args.Length < 1)
+                    throw new InvalidOperationException($"must be called with exactly 1 string argument");
```

Comment:
Is this true? Can't we call time series function with more than one arg? 
What is we want to pass additional parameters?

Reply:
no, this is wrong
this error message was copy & pasted from somewhere else
need to write proper error messages